### PR TITLE
Feat/comment

### PR DESCRIPTION
**Pull request by imanmalekian31**
New Comment component.

### CODE REVIEW

File: src/components/unstyled/comment.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
   @apply flex items-center py-2 px-4 font-light;
 
   &-message {
-    @apply flex flex-col pl-2;
+    @apply flex flex-col;
```
Comment (PadowYT2) [inline src/components/unstyled/comment.css:5]: This was a bad change, without it, the image and the text are linked

**Review by PadowYT2 — CHANGES_REQUESTED** [review]

**Comment by taylorjdawson** [comment]
@imanmalekian31 @PadowYT2 what is the status on this?

**Comment by imanmalekian31** [comment]
@taylorjdawson, @saadeghi should answer it :)

**Comment by PadowYT2** [comment]
And I'm not a dev, just a user

**Comment by saadeghi** [comment]
It's on hold. We need to be careful.  
I know it's exciting to get new components faster but of we don't consider all use cases for a component, we will have unnecessary breaking changes, forcing everyone to change their HTML, and that's not good at all.  
I hope you understand and give it time because reviewing PRs like this is not just reviewing the code. It's about design and testing it will all possible common use cases a designer may think of for that specific component.

**Comment by saadeghi** [comment]
Moving forward with version `3.0` I have no plans for adding a `comment` class for now. There are many designs that can be implemented and we need to be flexible with that however the main difference between different designs of a comment box is mostly the grid/flex layout which is already easy to make using Tailwind utility classes.

Thanks for the PR

**Pull request closed without merging**